src/main/resources/templates/layouts/baseLayout.html

```diff
@@ -7,34 +7,38 @@
   <title layout:title-pattern="$CONTENT_TITLE"></title>
   <!--[if lte IE 8]>
   <link
-      href="@{{cdnUrl}/stylesheets/govuk-frontend/v2.12.0/govuk-frontend-ie8-2.12.0.min.css(cdnUrl=${@environment.getProperty('cdn.url')})}"
-      rel="stylesheet" type="text/css"/><![endif]-->
-
+      href="@{{cdnUrl}/stylesheets/govuk-frontend/v3.11.0/govuk-frontend-ie8-3.11.0.min.css(cdnUrl=${@environment.getProperty('cdn.url')})}"
+      rel="stylesheet"
+      type="text/css"/>
+  <![endif]-->
   <link
       media="all"
       rel="stylesheet"
-      th:href="@{{cdnUrl}/stylesheets/govuk-frontend/v2.12.0/govuk-frontend-2.12.0.min.css(cdnUrl=${@environment.getProperty('cdn.url')})}"
+      th:href="@{{cdnUrl}/stylesheets/govuk-frontend/v3.11.0/govuk-frontend-3.11.0.min.css(cdnUrl=${@environment.getProperty('cdn.url')})}"
       type="text/css"/>
-
-  <link rel="stylesheet" th:href="@{/css/application.css}"
+  <link rel="stylesheet"
+        th:href="@{/css/application.css}"
         type="text/css"/>
```

Comment:
this shows up on two lines, it might be a GitHub display thing but it could be an auto formatting thing

Reply:
I realised that its formatted like this elsewhere